pkg/dispatcher/dispatcher.go

```diff
@@ -163,8 +158,8 @@ func (d *Dispatcher) dispatch(ctx context.Context, msg wabbit.Delivery, ceClient
 
 	response, result := ceClient.Request(ctx, event)
 	if !isSuccess(ctx, result) {
-		logging.FromContext(ctx).Warnf("Failed to deliver to %q requeue: %v", d.SubscriberURL, d.Requeue)
-		err = msg.Nack(ackMultiple, d.Requeue)
+		logging.FromContext(ctx).Warnf("Failed to deliver to %q", d.SubscriberURL)
+		err = msg.Nack(ackMultiple, false)
 		if err != nil {
```

Comment:
Could we merge these two lines to `if err = msg.Nack(ackMultiple, d.Requeue); err != nil {` ?